bolt-design-system/bolt
# Button/Link | Bug | Double focus styles on button and link

## Jira

http://vjira2:8080/browse/BDS-1539

## Summary

Remove double-outline from `bolt-button` and `bolt-link` on focus.

## Details

These components delegate focus to the inner element. Turn off outline on the custom-element itself so that we don't get double, extra-heavy outline.

## How to test
1. View these pages on master:
    - https://master.boltdesignsystem.com/pattern-lab/patterns/02-components-button-05-button/02-components-button-05-button.html
    - https://master.boltdesignsystem.com/pattern-lab/patterns/02-components-link-05-link/02-components-link-05-link.html
2. Tab into the button/link element so that it has focus and note the outline style. It is very dark blue.
3. Run this branch locally and do the same. You should see the outline is 50% lighter.
